pkg/util/utils.go

```diff
@@ -260,7 +261,9 @@ func FindPlatform(name string) []string {
 	if openstackRegex.MatchString(name) {
 		platforms = append(platforms, "openstack")
 	}
-	if metalRegex.MatchString(name) {
+	if metalIPIRegex.MatchString(name) {
+		platforms = append(platforms, "metal-ipi")
+	} else if metalRegex.MatchString(name) {
```

Comment:
Does the fall through make sense? So a metal job is selected as ipi or not. I could make the metal regex more robust if this is too unclear

Reply:
yeah i'd rather see you make the existing metal regex more robust(mutually exclusive with the new one), lest this get reordered at some point in the future and screw it up.

Reply:
Hm, problem is golang doesn't do negative lookaheads in the regexp libarary, I'll have to play around with it.

There's also a bunch of problems that need fixing... metal-ipi is the first multi-stage job we've turned release informing on for, the job names are reversed because we use variants in openshift/release to create the periodic, so it ends up being named `ocp-4.5-e2e-metal-ipi`, but the job search wants the release as a suffix.

Also it's trying to query `operator.Run multi-stage test e2e-metal-ipi`, but you get no results in the search dashboard unless you remove `operator.`

Reply:
if there's no easy way to disambiguate the non-ipi jobs, we can just do it as you have it, just please add a comment on both the regex and the if/else making it very clear what the behavior is and why, so someone doesn't refactor it unknowingly.

regarding the other issues, let's chat on slack

Reply:
Updated, and sent you a message about the other issues.